.NET/Microsoft.Recognizers.Definitions.Common/Microsoft.Recognizers.Definitions.Common.csproj

```diff
@@ -379,12 +391,27 @@
     </None>
     <None Update="Spanish\NumbersWithUnitDefinitions.tt">
     </None>
+    <None Update="Swedish\ChoiceDefinitions.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+    </None>
+    <None Update="Swedish\NumbersDefinitions.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <CustomToolNamespace></CustomToolNamespace>
+    </None>
+    <None Update="Swedish\NumbersWithUnitDefinitions.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>NumbersWithUnitDefinitions.cs</LastGenOutput>
+    </None>
   </ItemGroup>
 
   <ItemGroup>
     <DotNetCliToolReference Include="T5.TextTransform.Tool" Version="1.1.0-*" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
```

Comment:
What is this service?

Reply:
When adding at .tt file VS adds this reference by default.

Reply:
Fixed (removed)